### PR TITLE
Corrects mistakes in security testing chapter.

### PR DESCRIPTION
**Pull request by Kyeraa**
Mostly fixes grammatical mistakes.
Importantly, changed the definitions of soundness and completeness to 'if' statements instead of 'iff' statements, since and 'if and only if' would imply that soundness and completeness are equivalent.

### CODE REVIEW

File: chapters/non-functional-testing/security-testing.md [hunk 1]
```diff
@@ -195,13 +195,13 @@ You should already be familiar with _white/black-box_ testing, static testing an
...
 
 ### Quality assessment criteria
 The quality of testing tools is evaluated in a number of ways. You have already learnt about _code coverage_ in a previous chapter. Here, we discuss four new metrics that are often used in the context of security testing:
 Designing an ideal testing tool requires striking a balance between two measures: (a) Soundness, and (b) Completeness.
 
-**Soundness** dictates that there should be no False Negatives (FN) — no vulnerability should be skipped. This implies that no alarm is raised *IFF* there is no existing vulnerability in the *System Under Test (SUT)*. **Completeness** dictates that there should be no False Positives (FP) — no false alarm should be raised. This implies that an alarm is raised *IFF* a valid vulnerability is found.
+**Soundness** dictates that there should be no False Negatives (FN) — no vulnerability should be skipped. This implies that no alarm is raised *IF* there is no existing vulnerability in the *System Under Test (SUT)*. **Completeness** dictates that there should be no False Positives (FP) — no false alarm should be raised. This implies that an alarm is raised *IF* a valid vulnerability is found.
```
Comment (azqa) [inline chapters/non-functional-testing/security-testing.md:205]: Actually IFF is the correct format here. There are two definitions and both have their own IFF contexts. For soundness, we say: no alarm IFF no existing vulnerability. 
For completeness, an alarm raised IFF a valid vulnerability is found.

**Comment by azqa** [comment]
Hi Kyeraa, 

Thanks for the PR. There's one thing I've asked you to undo (see my comment above) and then we can merge it.

File: chapters/non-functional-testing/security-testing.md (hunk 1 above)
Comment (azqa) [reply chapters/non-functional-testing/security-testing.md:205]: There's also another issue in fuzzing chapter, section "Telemetry as heuristics". 
The last sentence says "reducing its the run-time" -> "reducing its run-time"

Comment (Kyeraa) [reply chapters/non-functional-testing/security-testing.md:205]: I still cannot quite follow. I understand the passage as follows. 
Take 'a' = "an alarm is raised"
and 'v' = "there exists a vulnarability"
Then the passage is giving the following definitions:
1) Soundness = ¬a <-> ¬v (there is no alarm if and only if (iff) there is no vulnerability)
2) Completeness = a <-> v (there is an alarm if and only if (iff) there is a vulnerability)
These two statements are logically equivalent.
Instead, "no false negatives" translates to v -> a or ¬a -> ¬v
and "no false positives" translates to a -> v or ¬v -> ¬a
Combining these two, i.e. having something both sound and complete, would then yield  a <-> v or ¬a <-> ¬v: the ideal state in which there being an alarm or not always correspond to there being a vulnerability or not.

Hence, correct descriptions would be "if there is a vulnerability, an alarm is raised" (sound) and "if an alarm is raised, there is a vulnerability" (complete).
(Which is not how I corrected it here, but how I think it should be.)

**Comment by Kyeraa** [comment]
Thank you for your reaction, I commented above

File: chapters/non-functional-testing/security-testing.md (hunk 1 above)
Comment (azqa) [reply chapters/non-functional-testing/security-testing.md:205]: You're right. I misunderstood earlier. They have an implies relation, not IFF relation.

**Comment by mauricioaniche** [comment]
Just waiting for @azqa's LGTM!

**Comment by azqa** [comment]
@mauricioaniche  LGTM!

**Pull request merged**